### PR TITLE
Added intalaltion timeout to config

### PR DESCRIPTION
**Pull request by eranco74**
https://bugzilla.redhat.com/show_bug.cgi?id=1862039

### CODE REVIEW

File: src/config/config.go [hunk 1]
```diff
@@ -40,6 +41,7 @@ func ProcessArgs() {
 	flag.BoolVar(&ret.Verbose, "verbose", false, "Increase verbosity, set log level to debug")
 	flag.StringVar(&ret.ControllerImage, "controller-image", "quay.io/ocpmetal/assisted-installer-controller:latest",
 		"Assisted Installer Controller image URL")
+	flag.IntVar(&ret.InstallationTimeout, "installation-timeout", 120, "Installation timeout in minutes")
```
Comment (filanov) [inline src/config/config.go:44]: you can use `flag.DurationVar`

File: src/config/config.go [hunk 2]
```diff
@@ -40,6 +42,7 @@ func ProcessArgs() {
 	flag.BoolVar(&ret.Verbose, "verbose", false, "Increase verbosity, set log level to debug")
 	flag.StringVar(&ret.ControllerImage, "controller-image", "quay.io/ocpmetal/assisted-installer-controller:latest",
 		"Assisted Installer Controller image URL")
+	flag.DurationVar(&ret.InstallationTimeout, "installation-timeout", 120, "Installation timeout in minutes")
```
Comment (filanov) [inline src/config/config.go:45]:
```suggestion
	flag.DurationVar(&ret.InstallationTimeout, "installation-timeout", 120 * time.Minutes, "Installation timeout in minutes")
```

Comment (filanov) [reply src/config/config.go:45]: otherwise it will be miliseconds

Comment (eranco74) [reply src/config/config.go:45]: https://github.com/eranco74/assisted-installer/pull/89/files#diff-8bea30c4cde4eea8efce766b4de08e43R281
I want to user to specify minutes.

Comment (filanov) [reply src/config/config.go:45]: then it's probably should be IntVal it will be clearer

**Pull request merged**